hypertrace-trace-enricher/enriched-span-constants/build.gradle.kts

```diff
@@ -63,13 +63,14 @@ sourceSets {
 }
 
 dependencies {
-  api("com.google.protobuf:protobuf-java-util:3.17.3")
+  api("com.google.protobuf:protobuf-java-util:3.21.1")
 
   implementation("org.hypertrace.core.datamodel:data-model:0.1.22")
   implementation(project(":span-normalizer:raw-span-constants"))
   implementation(project(":span-normalizer:span-normalizer-constants"))
   implementation(project(":semantic-convention-utils"))
   implementation("org.hypertrace.entity.service:entity-service-api:0.8.5")
+  implementation("com.google.guava:guava:31.1-jre")
```

Comment:
Went through and declared an explicit guava dependency in every project directly using it. We shouldn't be depending on transitives - doing so here left us vulnerable to that transitive version changing and breaking us (which is how I noticed).